### PR TITLE
Add Prometheus Operator Helm chart

### PR DESCRIPTION
**Pull request by eduartua**

### CODE REVIEW

**Comment by victortrac** [comment]
@eduartua This PR has conflicting terraform changes that need to be addressed.

**Comment by eduartua** [comment]
All conflict have been resolved.

File: kubernetes/helm/prometheus/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v2
+name: prometheus
+description: Prometheus operator
+version: 0.1.0
+appVersion: 0.1.0q
```
Comment (victortrac) [inline kubernetes/helm/prometheus/Chart.yaml]: This appVersion seems wrong.

Comment (eduartua) [reply kubernetes/helm/prometheus/Chart.yaml]: Yeah, that q was when doing something with vim. I'll fix that.

**Pull request merged**